src/lancie-admin-layout/lancie-admin-layout.html

```diff
@@ -95,6 +114,13 @@
         }
       },
 
+      attached: function() {
+        this.async(function() {
+          this.$.drawerLayout.resetLayout();
+          this.$.headerLayout.resetLayout();
```

Comment:
Yeah, this is the quick and dirty version of doing this. Don't know how to fix the layout thingy...